package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "module",
+  "name": "call_of_duty",
+  "version": "1.0.0",
+  "description": "Call Of Duty",
+  "main": "server.js",
+  "scripts": {
+    "test": "vitest --run --watch --silent --threads false",
+    "start": "node server.js",
+    "dev": "nodemon server.js",
```

Comment:
Just for info - no need to fix:
node have watch mod now so no need for `nodemon`. you can use `node -w`

Reply:
thanks :)